### PR TITLE
[Gaskrank] fix for broken site.

### PR DESCRIPTION
**Pull request by motophil**
### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)

### What is the purpose of your *pull request*?
- [x] Bug fix

---

Fix for broken site gaskrank.tv

### CODE REVIEW

File: youtube_dl/extractor/gaskrank.py [hunk 1]
```diff
@@ -89,23 +85,22 @@ def fix_json(code):
...
-            display_id, transform_source=fix_json, fatal=False)
-
         video_id = self._search_regex(
             r'https?://movies\.gaskrank\.tv/([^-]*?)(-[^\.]*)?\.mp4',
-            playlist.get('0').get('src'), 'video id')
+            webpage, 'video id')
+
+        playlist = re.findall(
+            r'<source([^>]+)',
+            webpage)
```
Comment (dstftw) [inline youtube_dl/extractor/gaskrank.py]: Use `self._parse_html5_media_entries` instead.

Comment (motophil) [inline youtube_dl/extractor/gaskrank.py]: Fixed.

File: youtube_dl/extractor/gaskrank.py [hunk 2]
```diff
@@ -89,35 +82,25 @@ def fix_json(code):
...
-            'categories': categories,
-            'display_id': display_id,
-            'uploader_id': uploader_id,
-            'upload_date': upload_date,
-            'uploader_url': uploader_url,
-            'tags': tags,
-            'view_count': view_count,
-            'average_rating': average_rating,
-        }
+        return self.playlist_result(entries)
```
Comment (dstftw) [inline youtube_dl/extractor/gaskrank.py]: There is always only one video on the page thus you should take first entry and return as regular info dict.

File: youtube_dl/extractor/gaskrank.py [hunk 3]
```diff
@@ -49,19 +48,13 @@ class GaskrankIE(InfoExtractor):
...
-        def fix_json(code):
-            """Removes trailing comma in json: {{},} --> {{}}"""
-            return re.sub(r',\s*}', r'}', js_to_json(code))
 
         display_id = self._match_id(url)
         webpage = self._download_webpage(url, display_id)
         categories = [re.match(self._VALID_URL, url).group('categories')]
         title = self._search_regex(
-            r'movieName\s*:\s*\'([^\']*)\'',
+            r'<meta[^>]+itemprop\s*=\s*"name"[^>]+content\s*=\s*"([^"]+)"',
```
Comment (dstftw) [inline youtube_dl/extractor/gaskrank.py]: Use `self._html_search_meta` or `self._og_search_title` instead.

Comment (motophil) [inline youtube_dl/extractor/gaskrank.py]: Fixed.

File: youtube_dl/extractor/gaskrank.py (hunk 2 above)
Comment (motophil) [inline youtube_dl/extractor/gaskrank.py]: Fixed.

**Pull request closed without merging**